### ISSUE TITLE
Upgrade Kusama and Polkadot runtimes to use MessageQueue Pallet 

### ISSUE THREAD

**Issue by claravanstaden** [polkadot-fellows/runtimes#162]
On the Rococo, the DMP Queue pallet was removed and the [MessageQueue pallet was configured so that XCMP](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L362) is used. The same needs to be done for Kusama and Polkadot. This is a prerequisite for the Snowbridge PR: https://github.com/polkadot-fellows/runtimes/pull/130

cc @ggwpez @vgeddes @musnit @alistair-singh @ryong @acatangiu

**Comment by bkontur** [comment]
yes, I've made a comment for you [here](https://github.com/polkadot-fellows/runtimes/pull/130#issuecomment-1905954969) few days ago.
Here is a bump to [`polkadot-sdk@1.5.0`](https://github.com/polkadot-fellows/runtimes/pull/137) where MQ has been introduced for BridgeHubs [here](https://github.com/polkadot-fellows/runtimes/pull/137/files#diff-da00a6055ce091ceb6769ed5cc0dc6527aca586de0394a60aa3d18c38ff89716R320).
And [here](https://github.com/polkadot-fellows/runtimes/pull/159) is another bump (not stable now) to `polkadot-sdk@1.6.0` with MQ re-entrancy.


so I would suggest to rebase https://github.com/polkadot-fellows/runtimes/pull/130 above one of those bumps to get closer.

**Comment by bkontur** [comment]
@claravanstaden Can we close this now?

**Comment by ggwpez** [comment]
Yes the MQ pallet is on Polkadot and Kusama now.
